### PR TITLE
[#151674775] Architecture documentation

### PR DESCRIPTION
**Pull request by cloudify**

### CODE REVIEW

**Comment by cloudify** [comment]
@GiorgioNatili @gvarisco @gunzip can you start looking at the `Application architecture` ? In the meanwhile I'll work on the `Integration architecture` section, thx

File: README.md [hunk 1]
```diff
@@ -153,6 +154,100 @@ The business logic is designed to be purely functional and segregated from
...
+### Data architecture
+
+The data model is composed of the following core entities:
+
+|Entity|Description|
+|-|-|
+|[Profile](https://github.com/teamdigitale/digital-citizenship-functions/blob/master/lib/models/profile.ts)|Represents the profile of a citizen (i.e. his preferences)|
+|[Message](https://github.com/teamdigitale/digital-citizenship-functions/blob/master/lib/models/message.ts)|Represents a message sent to a citizen by a service owned by an Organization|
+|[Notification](https://github.com/teamdigitale/digital-citizenship-functions/blob/master/lib/models/notification.ts)|Represents a notification to the citizen, triggered by a Message|
+|[Organization](https://github.com/teamdigitale/digital-citizenship-functions/blob/master/lib/models/organization.ts)|Represents an Organization whos services call the APIs to sent messages or access profiles|
```
Comment (gunzip) [inline README.md]: whose ?

Comment (gunzip) [inline README.md]: mayeb better: "Represents the Organization that send the messages or access citizens profile using the APIs"

Comment (cloudify) [reply README.md]: 👍

**Pull request merged**